src/clj/vanilla/subscription_manager.clj

```diff
@@ -27,12 +28,17 @@
   ;(prn "add-empty-user " username)
   (swap! subscribed-sources assoc-in [:subscribers username] {:username username :sources #{}}))
 
+(defn clean-sources
+  "Verifies that all sources are valid and removes any invalid ones"
+  [sources]
+  (into [] (remove #(nil? (get-servicedep %)) sources)))
+
 
 
 (defn push-data!
   "Gets the websocket send-fn from the system-map to push out the updated data-source"
   [username source]
-  (prn "push-data! " username source)
+  (log/info "push-data! " username source)
```

Comment:
do we really want to log every time data is pushed to a user?

Reply:
I was thinking of keeping this in at least for a bit so we have some logging thats viewable from the AWS console and gives a view into whats happening when its running there.